x/wasm/client/cli/genesis_msg.go

```diff
@@ -71,6 +71,7 @@ func GenesisStoreCodeCmd(defaultNodeHome string, genesisMutator GenesisMutator)
 	}
 	cmd.Flags().String(flagRunAs, "", "The address that is stored as code creator")
 	cmd.Flags().String(flagInstantiateByEverybody, "", "Everybody can instantiate a contract from the code, optional")
+	cmd.Flags().String(flagInstantiateNobody, "", "Nobody can instantiate a contract from the code, optional")
```

Comment:
Would almost want to call it `flagInstantiateGovernace`, `"Only governance can instantiate a contract from the code, optional"`. That might be more clear?

Otherwise this is awesome!

Reply:
this follows the naming convention of  `AcessConfig` while it's true only governance will be able to instantiate it with a default wasmd setup, might not be the case for other chains using the keeper directly with different `AuthorizationPolicy`.  Happy to change it tho to reflect the most common case.

Reply:
Maybe, `instantiate-governance-only` ?

Reply:
The convention so far has been to use `nobody` no?

Reply:
I like being verbose in the description. Without more context "nobody" sounds odd. 
How about "Nobody except the governance process can instantiate a contract from the code, optional" 🤷

Reply:
updated description